src/Compiler/AbstractIL/il.fs

```diff
@@ -2781,6 +2781,8 @@ type ILTypeDef
 
     member _.MetadataIndex = metadataIndex
 
+    member _.Flags = additionalFlags
```

Comment:
This is expected, basic APIs will likely have to increase their surface to make them accessible to the new pickling code.